src/app/story/test.data.ts

```diff
@@ -535,21 +535,21 @@ export const greenwaveDecision: GreenwaveDecision = {
     {
       'data': {
         'item': [
-          'cfme-openshift-app-ui-container-5.9.3.4-1.1533127933'
+          'python36-3.6-8010020190626162728.a920e634'
         ],
         'log': [
-          'https://jenkins.domain.local/job/cvp-product-test/1/console'
+          'https://jenkins.domain.local/job/ci-openstack-mbs/120/console'
         ],
       },
       'groups': [
-        'b6d8bbeb-a7ea-4bc3-b062-25eb8f404b2e'
+        '7e03a6b6-6341-4826-ac35-9293bc4c851d'
       ],
-      'href': 'https://resultsdb.domain.local/api/v2.0/results/6125318',
-      'id': 6125318,
+      'href': 'https://resultsdb.domain.local/api/v2.0/results/7504231',
+      'id': 7504231,
       'note': '',
-      'outcome': 'PASSED',
```

Comment:
Now all of the outcomes are `INFO`. We should have at least one `PASSED`. There should also be a `FAILED` result that gets waived.